### PR TITLE
[LICENSE] Add more information for license

### PR DESCRIPTION
**Pull request by nbei**

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -177,3 +177,7 @@ c. 安装完整版并且编译 onnxruntime 的自定义算子
 ## 贡献指南
 
 我们感谢所有的贡献者为改进和提升 MMCV 所作出的努力。请参考[贡献指南](CONTRIBUTING.md)来了解参与项目贡献的相关指引。
+
+## 证书及版权问题
```
Comment (hellock) [inline README_zh-CN.md:181]: license: 许可证

File: README.md [hunk 2]
```diff
@@ -181,3 +181,7 @@ you may first refer to this [Trouble Shooting Page](https://mmcv.readthedocs.io/
 ## Contributing
 
 We appreciate all contributions to improve MMCV. Please refer to [CONTRIBUTING.md](CONTRIBUTING.md) for the contributing guideline.
+
+## License
+
+Some operations in `MMCV` are with other licenses instead of Apache2.0. Please refer to [LICENSES.md](LICENSES.md) for the careful check, if you are using our code for commercial matters.
```
Comment (hellock) [inline README.md:187]: MMCV is released under the Apache 2.0 license, while some specific operations in this library are with other licenses. Please refer to [LICENSES.md](LICENSES.md) for the careful check, if you are using our code for commercial matters.

File: README_zh-CN.md [hunk 3]
```diff
@@ -178,6 +178,11 @@ c. 安装完整版并且编译 onnxruntime 的自定义算子
 
 我们感谢所有的贡献者为改进和提升 MMCV 所作出的努力。请参考[贡献指南](CONTRIBUTING.md)来了解参与项目贡献的相关指引。
 
+<<<<<<< HEAD
```
Comment (hellock) [inline README_zh-CN.md:181]: clean these marks

File: mmcv/ops/csrc/pytorch/fused_bias_leakyrelu.cpp [hunk 4]
```diff
@@ -1,3 +1,100 @@
...
+OR RELATED TO THIS LICENSE, THE USE OR INABILITY TO USE THE WORK
+(INCLUDING BUT NOT LIMITED TO LOSS OF GOODWILL, BUSINESS INTERRUPTION,
+LOST PROFITS OR DATA, COMPUTER FAILURE OR MALFUNCTION, OR ANY OTHER
+COMMERCIAL DAMAGES OR LOSSES), EVEN IF THE LICENSOR HAS BEEN ADVISED OF
+THE POSSIBILITY OF SUCH DAMAGES.
+
+=======================================================================
+*/
 // Modified from
 // from
```
Comment (hellock) [inline mmcv/ops/csrc/pytorch/fused_bias_leakyrelu.cpp:99]: duplicated

File: mmcv/ops/fused_bias_leakyrelu.py [hunk 5]
```diff
@@ -1,3 +1,98 @@
...
+# INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES ARISING OUT OF
+# OR RELATED TO THIS LICENSE, THE USE OR INABILITY TO USE THE WORK
+# (INCLUDING BUT NOT LIMITED TO LOSS OF GOODWILL, BUSINESS INTERRUPTION,
+# LOST PROFITS OR DATA, COMPUTER FAILURE OR MALFUNCTION, OR ANY OTHER
+# COMMERCIAL DAMAGES OR LOSSES), EVEN IF THE LICENSOR HAS BEEN ADVISED OF
+# THE POSSIBILITY OF SUCH DAMAGES.
+
+# =======================================================================
+
 # modify from https://github.com/rosinality/stylegan2-pytorch/blob/master/op/fused_act.py # noqa:E501
```
Comment (hellock) [inline mmcv/ops/fused_bias_leakyrelu.py:96]: modified from

File: mmcv/ops/upfirdn2d.py [hunk 6]
```diff
@@ -1,3 +1,98 @@
...
+# INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES ARISING OUT OF
+# OR RELATED TO THIS LICENSE, THE USE OR INABILITY TO USE THE WORK
+# (INCLUDING BUT NOT LIMITED TO LOSS OF GOODWILL, BUSINESS INTERRUPTION,
+# LOST PROFITS OR DATA, COMPUTER FAILURE OR MALFUNCTION, OR ANY OTHER
+# COMMERCIAL DAMAGES OR LOSSES), EVEN IF THE LICENSOR HAS BEEN ADVISED OF
+# THE POSSIBILITY OF SUCH DAMAGES.
+
+# =======================================================================
+
 # modify from https://github.com/rosinality/stylegan2-pytorch/blob/master/op/upfirdn2d.py  # noqa:E501
```
Comment (hellock) [inline mmcv/ops/upfirdn2d.py:96]: modified from

File: README_zh-CN.md (hunk 3 above)
Comment (hellock) [reply README_zh-CN.md:181]: Add check-merge-conflict hook to prevent such things.

File: README_zh-CN.md [hunk 7]
```diff
@@ -178,6 +178,9 @@ c. 安装完整版并且编译 onnxruntime 的自定义算子
 
 我们感谢所有的贡献者为改进和提升 MMCV 所作出的努力。请参考[贡献指南](CONTRIBUTING.md)来了解参与项目贡献的相关指引。
 
+## 许可证
+
+在 `MMCV` 中，有一部分功能并不是使用的 Apache2.0 证书，我们在 [证书](LICENSES.md) 中详细的列出了这些功能已经他们对应的证书，如果您正在从事盈利性活动，请谨慎参考此文档。
```
Comment (hellock) [inline README_zh-CN.md:183]: The chinese version should be consistent with the english one.

**Comment by hellock** [comment]
Add `check-merge-conflict` hook to prevent merge conflicts.

File: mmcv/ops/csrc/pytorch/fused_bias_leakyrelu.cpp [hunk 8]
```diff
@@ -1,5 +1,101 @@
...
+(INCLUDING BUT NOT LIMITED TO LOSS OF GOODWILL, BUSINESS INTERRUPTION,
+LOST PROFITS OR DATA, COMPUTER FAILURE OR MALFUNCTION, OR ANY OTHER
+COMMERCIAL DAMAGES OR LOSSES), EVEN IF THE LICENSOR HAS BEEN ADVISED OF
+THE POSSIBILITY OF SUCH DAMAGES.
+
+=======================================================================
+*/
 // Modified from
-// from
 // https://github.com/rosinality/stylegan2-pytorch/blob/master/op/fused_bias_act.cpp
```
Comment (hellock) [inline mmcv/ops/csrc/pytorch/fused_bias_leakyrelu.cpp:99]: `Modified from xxx` can be put at the top of this file.

**Comment by ZwwWayne** [comment]
ping @nbei

File: README_zh-CN.md [hunk 9]
```diff
@@ -178,6 +178,9 @@ c. 安装完整版并且编译 onnxruntime 的自定义算子
 
 我们感谢所有的贡献者为改进和提升 MMCV 所作出的努力。请参考[贡献指南](CONTRIBUTING.md)来了解参与项目贡献的相关指引。
 
+## 许可证
+
+`MMCV` 目前以 Apache 2.0 的许可证发布，但是其中有一部分功能并不是使用的 Apache2.0 证书，我们在 [证书](LICENSES.md) 中详细的列出了这些功能已经他们对应的证书，如果您正在从事盈利性活动，请谨慎参考此文档。
```
Comment (hellock) [inline README_zh-CN.md:183]: 证书 -> 许可证

File: README_zh-CN.md [hunk 10]
```diff
@@ -180,7 +180,7 @@ c. 安装完整版并且编译 onnxruntime 的自定义算子
 
 ## 许可证
 
-`MMCV` 目前以 Apache 2.0 的许可证发布，但是其中有一部分功能并不是使用的 Apache2.0 证书，我们在 [证书](LICENSES.md) 中详细的列出了这些功能已经他们对应的证书，如果您正在从事盈利性活动，请谨慎参考此文档。
+`MMCV` 目前以 Apache 2.0 的许可证发布，但是其中有一部分功能并不是使用的 Apache2.0 许可证，我们在 [许可证](LICENSES.md) 中详细的列出了这些功能已经他们对应的许可证，如果您正在从事盈利性活动，请谨慎参考此文档。
```
Comment (ZwwWayne) [inline README_zh-CN.md:183]: 已经 -> 以及

**Comment by ZwwWayne** [comment]
ping @nbei

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**